modules/node_modules/@frogpond/ccci-stolaf-college/v1/calendar/index.js

```diff
@@ -55,6 +55,7 @@ export async function krlx(ctx) {
 export async function ksto(ctx) {
 	ctx.cacheControl(ONE_MINUTE)
 
-	let id = 'kstonarwhal@gmail.com'
+	// let id = 'kstonarwhal@gmail.com'
```

Comment:
I think this line should get removed, primarily because the fact that the solution here was to create a new calendar rather than update the old one is a sign that they will prefer this approach in the future.

As @drewvolz suggests, we do have Git.